apps/python-sdk/pyproject.toml

```diff
@@ -12,7 +12,9 @@ dependencies = [
     "requests",
     "python-dotenv",
     "websockets",
-    "nest-asyncio"
+    "nest-asyncio",
+    "pytest>=8.3.4",
```

Comment:
we might not need pytest here, @rafaelmmiller lmk.

Reply:
I have removed pytest from pyproject.toml but while testing i get this error, thats why I added it

![image](https://github.com/user-attachments/assets/e2aac6df-6073-4fdd-93c4-04c9a9f41a25)

Reply:
Oh i see